MetaMask/metamask-extension
# Fix typo in the name of the Polish language

## Explanation

This PR fixes typo in the name of the Polish language. 
Translation: https://translate.google.pl/?sl=en&tl=pl&text=Polish&op=translate

## Manual Testing Steps

- Open MetaMask extension
- Go to Settings -> General -> Language
- Check Polish language name (it should be "Polski")

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
